client/src/partials/journal/journal.html

```diff
@@ -17,6 +17,9 @@
   </div>
 </div>
 
+<!-- util bar  -->
+<div class="flex-util"></div>
```

Comment:
Ah, I see what you did.  So, every grid is required to have a util bar?  This seems like a poor solution...

Maybe you could make two classes?